src-backend/api/serializer.py

```diff
@@ -17,7 +17,9 @@ class Meta:
             'page',
             'choices',
             'numeric',
-            'element_type'
+            'element_type',
+            'last_modified',
```

Comment:
maybe these should be worded `last_modified_at` and `created_at` to follow how Rails does it